valhalla/baldr/graphreader.h

```diff
@@ -343,6 +343,12 @@ class GraphReader {
   explicit GraphReader(const boost::property_tree::ptree& pt,
                        std::unique_ptr<tile_getter_t>&& tile_getter = nullptr);
 
+  void set_tile_getter_interrupt(const tile_getter_t::interrupt_t* interrupt) {
```

Comment:
The naming style is a bit off compared to graph reader class (which is more of pure google c++ code style)

Reply:
oh I see, thanks for pointing me out
though the code style of namespace baldr is a little bit confusing :/